cmd/machine-config-operator/bootstrap.go

```diff
@@ -19,23 +19,28 @@ var (
 	}
 
 	bootstrapOpts struct {
-		cloudConfigFile      string
-		configFile           string
-		destinationDir       string
-		etcdCAFile           string
-		etcdImage            string
-		etcdMetricCAFile     string
-		imagesConfigMapFile  string
-		infraConfigFile      string
-		infraImage           string
-		kubeCAFile           string
-		kubeClientAgentImage string
-		mcoImage             string
-		networkConfigFile    string
-		oscontentImage       string
-		pullSecretFile       string
-		rootCAFile           string
-		proxyConfigFile      string
+		baremetalRuntimeCfgImage string
```

Comment:
Referencing Clayton's comment about naming https://github.com/openshift/machine-config-operator/pull/795#discussion_r303531723

Are we clear on this name code wise as well?

Reply:
yes it was resolved in https://github.com/openshift/release/pull/4364